Src/Fido2.AspNet/NullMetadataService.cs

```diff
@@ -15,7 +16,7 @@ MetadataBLOBPayloadEntry IMetadataService.GetEntry(Guid aaguid)
             return null;
         }
 
-        Task IMetadataService.InitializeAsync()
+        Task IMetadataService.InitializeAsync(CancellationToken cancellationToken)

```

Comment:
Make token optional

Reply:
I think here it is not necessary. The method is in an internal class and the `default` is already defined on the interface. An external consumer cannot invoke the method directly.

Reply:
Ok